common/Makefile.common

```diff
@@ -65,7 +65,7 @@ ${BOARD_BUILDDIR}/${TOP}.fasm: ${BOARD_BUILDDIR}/${TOP}.route
 	cd ${BOARD_BUILDDIR} && symbiflow_write_fasm -e ${TOP}.eblif -d ${DEVICE}
 
 ${BOARD_BUILDDIR}/${TOP}.bit: ${BOARD_BUILDDIR}/${TOP}.fasm
-	cd ${BOARD_BUILDDIR} && symbiflow_write_bitstream -d ${BITSTREAM_DEVICE} -f ${TOP}.fasm -p ${PARTNAME} -b ${TOP}.bit
+	cd ${BOARD_BUILDDIR} && symbiflow_write_bitstream -d ${BITSTREAM_DEVICE} -f ${TOP}.fasm -p ${PARTNAME} -b top.bit
```

Comment:
@acomodi I changed the common makefile to generate top.bit instead of relying on the TOP variable because the new instructions tell the user to use the common Makefile as part of their designs. If the name of the output bit stream is based on the name of the top level module variable than some users might set their TOP variable to something other than "top" and then wonder why the included openocd command (which assumes that the bit file is named top.bit) dosen't work. All of the examples in symbiflow-examples assume that ``symbiflow_write_bitstream`` generates a file named top.bit and I think it would be better for the included makefile to just always generate this file for consistency.

Reply:
I think that we should leave the ability to change the bitstream name and that the change should not happen here, but rather in removing the assumption that `symbiflow_write_bitstream` generates a file named top.bit.

Reply:
I have changed it back to ``${TOP}`` and put a note before the openocd command to change ``top.bit`` to whatever the name of their top level module is.